TLSharp.Tests.NUnit.old/TLSharp.Tests.NUnit.csproj

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
why are you adding these files with the .old extension? just delete the old stuff

Reply:
Ok, please ask someone to test it, and if it works well, remove the .old files. Then after that we can merge this PR.

Reply:
@seriouz can you test it?